### PR TITLE
fix(mp) blockchain Packs with no monitoring procedure

### PR DESCRIPTION
**Pull request by Sims24**
## Description

Adding missing monitoring procedure resulting in 403 errors. 

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* Apply the *Blockchain-Hyperledger-Exporter-custom* template and configure all the mandatory Macros:
+
+| Mandatory | Name               | Description                                                                        |
+|:----------|:-------------------|:-----------------------------------------------------------------------------------|
+|    x      | EXPORTERAPIPORT    | Port used by the Hyperledger Exporter (Default: '80')                              |
+|    x      | EXPORTERPROTO      | Protocol used by the Hyperledger Exporter (Default: 'http')                        |
+|    X      | EXPORTERAPIURLPATH | URL to access the Hyperledger Exporter (Default: '/')                              |
+|           | TIMEOUT            | Timeout (Default: '10')                                                            |
+|           | EXTRAOPTIONS       | Any extra option you may want to add to every command\_line (eg. a --verbose flag) |
+
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md:81]: We don't show an example for that plugin ?
the "How to test in CLI" part ?

File: en/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+All available options for a given mode can be displayed by adding the 
+```--help``` parameter to the command:
+
+ All plugin modes can be listed with the following command:
+
+ ```bash
+ /usr/lib/centreon/plugins//centreon_blockchain_parity_ethpoller_api.pl  \ 
+    --plugin=blockchain::parity::ethpoller::plugin  \ 
+    --mode=tracking  \ 
+    --help
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md:155]:
```suggestion
    --list-mode
```

File: en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+| Metric name                   | Description              |
+|:------------------------------|:-------------------------|
+| parity.network.peers.count    | Number of known peers    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To get data from the API, the Poller must be able to communicate with it over the 
+configured port (default: 8545).
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:63]: No documentation available about "how to enable the API" ?

File: fr/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+* Appliquez le Modèle d'Hôte *Blockchain-Hyperledger-Exporter-custom*
+
+| Mandatory | Name               | Description                                                                        |
+|:----------|:-------------------|:-----------------------------------------------------------------------------------|
+|    x      | EXPORTERAPIPORT    | Port used by the Hyperledger Exporter (Default: '80')                              |
+|    x      | EXPORTERPROTO      | Protocol used by the Hyperledger Exporter (Default: 'http')                        |
+|    X      | EXPORTERAPIURLPATH | URL to access the Hyperledger Exporter (Default: '/')                              |
+|           | TIMEOUT            | Timeout (Default: '10')                                                            |
+|           | EXTRAOPTIONS       | Any extra option you may want to add to every command\_line (eg. a --verbose flag) |
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md:80]: see my english comment

File: fr/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+## Diagnostic des erreurs communes
+
+### UNKNOWN: Can't connect to ... 
+
+Cette erreur signifie que Centreon n'a pas réussi à se connecter à l'API du 
+serveur Ciso ISE. Vérifiez que la requête n'est pas bloquée par un outil externe
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md:86]:
```suggestion
Cette erreur signifie que Centreon n'a pas réussi à se connecter à l'API de 
l'Hyperledger. Vérifiez que la requête n'est pas bloquée par un outil externe
```

File: fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
+---
+id: blockchain-parity-ethpoller-restapi
+title: Ethpoller API
+---
+
+## Vue d'ensemble
+
+Le BCM Poller expose collecte des données sur le comportement d'un réseau Blockchain
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md:8]: The BCM Poller ?

File: fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+| eth.poller.disk.size            |  Total size                   |  B   |
+| eth.poller.disk.used            |  Used space                   |  B   |
+| eth.poller.disk.usage           |  Disk usage                   |  %   |
+| eth.poller.blockchain.directory |  Size of blockchain directory |  B   |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Afin de récupérer des données, déployer le BCM Poller en suivant la documentation
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md:61]: Aaaaah okey. The BCM Poller right.

File: fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+Les options disponibles pour un mode donnée peuvent être affiché en ajoutant 
+l'option `--help` à la commande.
+
+Tous les modes d'un Plugin donné peuvent être listés au moyen de la commande suivante:
+
+```bash
+/usr/lib/centreon/plugins//centreon_blockchain_parity_ethpoller_api.pl  \ 
+   --plugin=blockchain::parity::ethpoller::plugin  \ 
+   --mode=tracking  \ 
+   --help
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md:156]:
```suggestion
   --list-mode
```

File: en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+This error message means that the Centreon Plugin couldn't successfully connect to the Parity API. Check that no third party
+device (such as a firewall) is blocking the request. 
+
+### UNKNOWN: Cannot decode json response
+
+This error message means that the Centreon Plugin couldn't successfully connect to the Parity API. Check that no third party
+device (such as a firewall) is blocking the request. 
+
+A proxy connection may also be necessary to connect to the API.
+This can be done by using this option in the command: ```--proxyurl='http://proxy.mycompany:8080'```.
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:165]: Same troubleshooting solutions for 2 error messages ?

File: en/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+### UNKNOWN: Can't connect to ... 
+
+This error message means that the Centreon Plugin couldn't successfully connect to the Ethpoller API. Check that no third party
+device (such as a firewall) is blocking the request. 
+
+### UNKNOWN: Cannot decode json response
+
+This error message means that the Centreon Plugin couldn't successfully connect to the Ethpoller API. Check that no third party
+device (such as a firewall) is blocking the request. A proxy connection may also be necessary to connect to the API.
+This can be done by using this option in the command: ```--proxyurl='http://proxy.mycompany:8080'```.
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md:169]: Same troubleshooting solutions for 2 error messages ?

File: fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+### UNKNOWN: Can't connect to ... 
+
+Cette erreur signifie que Centreon n'a pas réussi à se connecter à l'API du 
+BCM Poller. Vérifiez que la requête n'est pas bloquée par un outil externe
+(un pare-feu par exemple). Si vous utilisez un proxy, renseignez son URL dans la
+Macro EXTRAOPTIONS de l'Hôte ou directement dans la commande avec l'option 
+```--proxyurl='http://proxy.mycompany:8080'```.
+
+Vérifiez également que le port configuré est correct.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:158]:
```suggestion
Vérifiez également que le port configuré est correct.

### J'obtiens le message d'erreur suivant:  ``UNKNOWN: 501 Protocol scheme 'connect' is not supported |```
Dans certains cas, et plus spécifiquement lors de l'usage d'un proxy 
d'entreprise, le protocole de connexion n'est pas supporté par la libraire lwp 
utlisée par défaut par le Plugin Centreon.
Cette erreur peut être résolue en utilisant le backend HTTP curl. Pour ce faire, 
ajoutez l'option ```--http-backend='curl'``` dans la Macro EXTRAOPTIONS de 
l'Hôte ou directement à la commande.
```

File: fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+
+### UNKNOWN: Can't connect to ... 
+
+Cette erreur signifie que Centreon n'a pas réussi à se connecter à l'API du 
+BCM Poller. Vérifiez que la requête n'est pas bloquée par un outil externe
+(un pare-feu par exemple). Si vous utilisez un proxy, renseignez son URL dans la
+Macro EXTRAOPTIONS de l'Hôte ou directement dans la commande avec l'option 
+```--proxyurl='http://proxy.mycompany:8080'```.
+
+Vérifiez également que le port configuré est correct.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md:169]:
```suggestion
Vérifiez également que le port configuré est correct.

### J'obtiens le message d'erreur suivant:  ``UNKNOWN: 501 Protocol scheme 'connect' is not supported |```
Dans certains cas, et plus spécifiquement lors de l'usage d'un proxy 
d'entreprise, le protocole de connexion n'est pas supporté par la libraire lwp 
utlisée par défaut par le Plugin Centreon.
Cette erreur peut être résolue en utilisant le backend HTTP curl. Pour ce faire, 
ajoutez l'option ```--http-backend='curl'``` dans la Macro EXTRAOPTIONS de 
l'Hôte ou directement à la commande.
```

File: fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+| Metric name                   | Description              |
+|:------------------------------|:-------------------------|
+| parity.network.peers.count    | Number of known peers    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Le Poller doit pouvoir communiquer avec le noeud Parity via le protocole HTTP et 
+via le port configuré (par défaut: 8545).
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:63]: See english comments

File: fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+Cette commande déclenchera une alerte WARNING si le nombre de peers connecté est 
+inférieur à 1 (`--critical-peers=1:`).
+
+Tous les modes d'un Plugin donné peuvent être listés au moyen de la commande suivante:
+
+```bash
+/usr/lib/centreon/plugins//centreon_blockchain_parity_api.pl  \ 
+    --plugin=blockchain::parity::restapi::plugin  \ 
+    --mode=net  \ 
+    --help
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:145]:
```suggestion
    --list-mode
```

**Review by pkriko — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md (hunk 1 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md:81]: No cause we don't have any sample command available.

File: en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md (hunk 3 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:63]: No, it's by default.

File: fr/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md (hunk 4 above)
Comment (Sims24) [reply fr/integrations/plugin-packs/procedures/blockchain-hyperledger-exporter.md:80]: Same answer

File: en/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md (hunk 10 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/blockchain-parity-ethpoller-restapi.md]: We will later link it to a dedicated page. But yeah, same cause, different effects :)

File: fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md (hunk 13 above)
Comment (Sims24) [reply fr/integrations/plugin-packs/procedures/blockchain-parity-restapi.md:63]: Same answer

**Review by pkriko — APPROVED** [review]

**Pull request merged**